odl/solvers/nonsmooth/primal_dual_hybrid_gradient.py

```diff
@@ -39,12 +40,12 @@ def chambolle_pock_solver(x, f, g, L, tau, sigma, niter, **kwargs):
 
     where ``L`` is an operator and ``F`` and ``G`` are functionals.
```

Comment:
The `F` and `G` references should all be lowercase, could you please fix that while at it?

Reply:
done